### ISSUE TITLE
Which network is ‘’features“？

### ISSUE THREAD

**Issue by lilidan-orm** [nxsEdson/MLCR#3]
In the file resnet_gcn, what does the "features" in "self. features. parameters ()" mean? I don't see the corresponding network. Thank you for your trouble.

**Comment by nxsEdson** [comment]
Sorry for the late reply and sorry that I forget to clean the useless lines in the file. However, it has no influence on the code.

**Comment by lilidan-orm** [comment]
> Sorry for the late reply and sorry that I forget to clean the useless lines in the file. However, it has no influence on the code.

Thank you very much. Now I use the natural images to train this model for multi-label task, but the value of  loss_similar is very very small (0.0x) from the beginning of training, is this a normal phenomenon?

**Comment by nxsEdson** [comment]
In my experiments, it takes about one or two epochs for the loss_similar to be small. I have adjusted the hyperparameters to increase the impact of loss_similar.
